### DISCUSSION TITLE
Feature request: Customize regconfig of Postgres by environment variable

### DISCUSSION THREAD

**Discussion by ykpre** [outline/outline#3750] [General]
Currently the config is hard coded 'english' like 'to_tsquery('english', :query))' and 'to_tsvector('english', :query)'. It would be great if Outline allow customizing the config by an environment variable.

Allow regconfig customization could be helpful for self-host users who use other Postgres extensions which support Chinese/Japanese tokenization. So they can create a new regconfig to solve the CJK search problem.

It could be a temporary workaround for #3207

**Comment by tommoor** [comment]
Would accept a PR for this, probably an optional env variable with `DATABASE_` prefix
